Makefile

```diff
@@ -389,16 +389,12 @@ ci-coverage-report: $(SUMMARY_COVER_PROFILE) coverage-report
 	@goveralls -coverprofile=$(SUMMARY_COVER_PROFILE) -service=buildkite || true
 
 ##### Schema #####
-install-schema: temporal-cassandra-tool
+install-schema-cass: temporal-cassandra-tool
```

Comment:
Can cass work alone? I mean without installing other schema?
If no, I think it will be better to install es schema here as well.

Reply:
Hm, sure, I can make it call `install-schema-es`.